npm/private/npm_linked_package_info.bzl

```diff
@@ -0,0 +1,14 @@
+"NpmLinkedPackageInfo provider"
+
+NpmLinkedPackageInfo = provider(
```

Comment:
nit: did you consider putting more than one provider in the same file?

Reply:
My preference is to have one file per provider. Makes the bar for adding a new provider a bit higher.